BiblioPixelSerialReceiver.ino

```diff
@@ -76,13 +76,22 @@ inline void setupFastLED()
     FastLED.show();
 }
 
+void fill(const struct CRGB & color){
```

Comment:
Cool, but no need for this... FastLED already has a fill function: http://fastled.io/docs/3.1/group___colorutils.html